src/uk/ac/ebi/vfb/neo4j/KB_tools.py

```diff
@@ -853,7 +853,7 @@ def __init__(self, endpoint, usr, pwd, use_base36=False):
             'depicts': 'http://xmlns.com/foaf/0.1/depicts',
             'in register with': 'http://purl.obolibrary.org/obo/RO_0002026',
             'is specified output of': 'http://purl.obolibrary.org/obo/OBI_0000312',
-            'hasDbXref': 'http://www.geneontology.org/formats/oboInOwl#hasDbXref',
+            'database_cross_reference': 'http://www.geneontology.org/formats/oboInOwl#hasDbXref',
```

Comment:
Looks like this is not being used in anywhere in code, so this is silent change (line should be deleted).